web/annotation/watermark annotations/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Watermark Annotations</title>
+  </head>
+  <body>
+    <script src="https://cdn.cloud.pspdfkit.com/pspdfkit-web@1.9.1/nutrient-viewer.js"></script>
```

Comment:
The version is hardcoded? How do we automatically update it, if we do that like in other repos?